dashboard/app/assets/stylesheets/DanceAiModal.scss

```diff
@@ -1,64 +1,15 @@
-/* This file generates CSS that will be used to animate the Blockly blocks
- * in the DanceAiModal's UI.  We need it to be CSS so that the styling can
- * be applied to Blockly blocks that we are embedding.  The CSS selectors
- * are highly-specific to the current version of Blockly, and to the blocks
- * that we are embedding, and will likely need to be updated in case any of
- * these things change.
- */
+/* This file generates CSS used to style an element in the DanceAiModal. */
 
-// Create an animation that delays for $delay seconds and then fades in
-// over the course of $duration seconds.
-@mixin appear($index) {
-  $name: appear-#{$index};
-  $delay: $index;
-  $duration: 1;
-
-  $start_percent: #{100 * ($delay / ($delay + $duration))};
-
-  @keyframes #{$name} {
-    0%, #{$start_percent}% {
-      opacity: 0;
-    }
-    100% {
-      opacity: 1;
-    }
-  }
-
-  animation: #{$name} #{$delay + $duration}s;
-}
+@import "color";
 
 #ai-modal-inner-area {
-  #fade-in {
-    // First block
-    .blocklyWorkspace .blocklyPath {
-      @include appear(0);
-    }
-
-    // First block's text
-    .blocklyBlockCanvas g .blocklyNonEditableText:nth-of-type(3) {
-      @include appear(1);
-    }
-
-    // Second block
-    .blocklyBlockCanvas g g .blocklyPath {
-      @include appear(2);
-    }
-
-    // Second block's first text
-    .blocklyBlockCanvas g .blocklyNonEditableText {
-      @include appear(3);
-    }
-
-    // Second block's second text
-    .blocklyBlockCanvas g .blocklyNonEditableText:nth-of-type(4) {
-      @include appear(4);
-    }
-  }
-
   #preview-area {
     #defaultCanvas1 {
       width: 100% !important;
       height: 100% !important;
+      border-radius: 4px;
+      border: 1px solid $neutral_dark30;
+      box-sizing: border-box;
```

Comment:
Can these not be set in `ai-visualization-preview.module.scss`? Are they getting overridden?

Reply:
Thanks for checking; I figured out how to make them work in that file.

Also, the other dimension overrides in this file no longer seem necessary, so this file is now gone entirely.